### DISCUSSION TITLE
Domain RNWL

### DISCUSSION THREAD

**Discussion by GreenCtzns** [vercel/community#7407] [Account Issue] [answered]
### Question

Good afternoon,

I hope this email finds you well. The system does not allow me to renew my domain. We are a company and we are deperate to fix the email problem and because our domain has expired we are unable to receive emails. How can we go about renewing the domain, please help!

**Comment by rafaelvf** [comment]
Im having the same problem. I cannot where to renew my domain.

**Reply by rafaelvf** [reply]
Did you recieve any help?

**Comment by pawlean** [comment]
Hi @GreenCtzns, @rafaelvf,

What issues are you seeing? 

Here is [our documentation](https://vercel.com/docs/projects/domains/renew-a-domain#renewing-a-domain-to-your-account) to help you renew your domain.

**Reply by rafaelvf** [reply]
Hi, I missed my domain renovation date, and now that I want to pay for the renewal its virtually impossible, I have look for all the dashboard and cannot find where to do it. I have created 2 support tickets but I have received any information.

**Reply by pawlean** [reply]
@rafaelvf Thanks for your patience, our team will get to you shortly.

**Reply by rafaelvf** [reply]
Thanks, where would I be contacted? here or directly in vercel?

**Reply by rafaelvf** [reply]
Hi I havent been contacted =(

**Reply by pawlean** [reply]
@rafaelvf I re-read your comment here. 

Can you try following the steps laid out in the documentation: https://vercel.com/docs/projects/domains/renew-a-domain#renewing-a-domain-to-your-account ?

**Reply by rafaelvf** [reply]
I did read this but I still cannot pay and renew it. That is my initial request.

**Reply by rafaelvf** [reply]
I have been contacted. Thanks

**Comment by GreenCtzns** [comment, marked as answer]
Hi,

On my behalf I did receive help! They sent me a link to where I was able to
make my payment.

Thank you.

Kind regards,

On Tue, Jul 9, 2024 at 10:26 AM Pauline P. Narvas ***@***.***>
wrote:

> Hi @GreenCtzns <https://github.com/GreenCtzns>, @rafaelvf
> <https://github.com/rafaelvf>,
>
> What issues are you seeing?
>
> Here is our documentation
> <https://vercel.com/docs/projects/domains/renew-a-domain#renewing-a-domain-to-your-account>
> to help you renew your domain.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/7407#discussioncomment-10000669>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBJ7HMJLCEBAUHMSYWAD27LZLQFMJAVCNFSM6AAAAABKRIY6SGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMBQGA3DMOI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
[image: created with MySignature.io]
Karla Baldioceda Fernández  Immigration
Green Citizens Legal | Costa Rica Law
e: ***@***.***
p: +(506) 8957-0556 <+(506)+8957-0556>
a: Liberia, Guanacaste
[image: created with MySignature.io]
<https://instagram.com/greencitizenslegalcostarica?igshid=YmMyMTA2M2Y=> [image:
created with MySignature.io]
<https://www.facebook.com/search/top?q=green%20citizens%20legal%20costa%20rica>
[image: created with MySignature.io]
<https://mysignature.io/editor/?utm_source=freepixel>

**Comment by GreenCtzns** [comment]
Hello,We payed through the link you sent I guess day a go it said that the Damian was going to be activated within 48 hours it has been longer than that and we are still unable to send email from our company email, because of the domain not been active we tried to follow the steps and we are still not able to activate it.Can you help! Best regards,Karla Baldioceda Fernández  ImmigrationGreen Citizens Legal | Costa ***@***.***: +(506) 8957-0556a: Liberia, Guanacaste El 11 jul 2024, a la(s) 12:45 p. m., RAFAEL VITERI ***@***.***> escribió:﻿
I did read this but I still cannot pay and renew it. That is my initial request.

**Comment by GreenCtzns** [comment]
Hello,

I hope this email finds you well. We made the payment through the link you
provided to us on Monday, the 8th. The confirmation indicated that the
domain would be activated within 48 hours. However, it has now been longer
than that, and we are still unable to send emails from our company email
because the domain is not active. We have followed the steps provided, but
we are still unable to activate it.

Can you help us?

Best regards,

On Thu, Jul 11, 2024 at 2:57 PM RAFAEL VITERI ***@***.***>
wrote:

> I have been contacted. Thanks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/7407#discussioncomment-10025209>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBJ7HMK6PPUJ7ENGXWJIBBLZL3WT7AVCNFSM6AAAAABKRIY6SGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMBSGUZDAOI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
[image: created with MySignature.io]
Karla Baldioceda Fernández  Immigration
Green Citizens Legal | Costa Rica Law
e: ***@***.***
p: +(506) 8957-0556 <+(506)+8957-0556>
a: Liberia, Guanacaste
[image: created with MySignature.io]
<https://instagram.com/greencitizenslegalcostarica?igshid=YmMyMTA2M2Y=> [image:
created with MySignature.io]
<https://www.facebook.com/search/top?q=green%20citizens%20legal%20costa%20rica>
[image: created with MySignature.io]
<https://mysignature.io/editor/?utm_source=freepixel>

**Comment by GreenCtzns** [comment]
Can you please help us??? We have already made the payment, we waited 48
hrs, and still are not able to receive or send emails from our accounts. Is
very important to our business to be able to answer emails as soon as
possible.  We do not know what else to do?? Please advise and help with a
solution.

Regards,

On Thu, Jul 11, 2024 at 4:04 PM Karla Baldioceda ***@***.***>
wrote:

> Hello,
>
> I hope this email finds you well. We made the payment through the link you
> provided to us on Monday, the 8th. The confirmation indicated that the
> domain would be activated within 48 hours. However, it has now been longer
> than that, and we are still unable to send emails from our company email
> because the domain is not active. We have followed the steps provided, but
> we are still unable to activate it.
>
> Can you help us?
>
> Best regards,
>
> On Thu, Jul 11, 2024 at 2:57 PM RAFAEL VITERI ***@***.***>
> wrote:
>
>> I have been contacted. Thanks
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/orgs/vercel/discussions/7407#discussioncomment-10025209>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BBJ7HMK6PPUJ7ENGXWJIBBLZL3WT7AVCNFSM6AAAAABKRIY6SGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMBSGUZDAOI>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>
>
> --
> [image: created with MySignature.io]
> Karla Baldioceda Fernández  Immigration
> Green Citizens Legal | Costa Rica Law
> e: ***@***.***
> p: +(506) 8957-0556 <+(506)+8957-0556>
> a: Liberia, Guanacaste
> [image: created with MySignature.io]
> <https://instagram.com/greencitizenslegalcostarica?igshid=YmMyMTA2M2Y=> [image:
> created with MySignature.io]
> <https://www.facebook.com/search/top?q=green%20citizens%20legal%20costa%20rica>
> [image: created with MySignature.io]
> <https://mysignature.io/editor/?utm_source=freepixel>
>
>
>
>
>

-- 
[image: created with MySignature.io]
Karla Baldioceda Fernández  Immigration
Green Citizens Legal | Costa Rica Law
e: ***@***.***
p: +(506) 8957-0556 <+(506)+8957-0556>
a: Liberia, Guanacaste
[image: created with MySignature.io]
<https://instagram.com/greencitizenslegalcostarica?igshid=YmMyMTA2M2Y=> [image:
created with MySignature.io]
<https://www.facebook.com/search/top?q=green%20citizens%20legal%20costa%20rica>
[image: created with MySignature.io]
<https://mysignature.io/editor/?utm_source=freepixel>

[a comment by an automated account is not shown]
